### ISSUE TITLE
cross experiments

### ISSUE THREAD

**Issue by yyjjylx** [zhengli97/PromptKD#36]
I had organized your datasets according to the instructions in the DATASET.md ,but there were still encountering errors when running cross-dataset experiments,
![1](https://github.com/user-attachments/assets/59e4e01d-8c35-4c40-b712-c2a4f451eea5)
![2](https://github.com/user-attachments/assets/89d997cd-305a-46b8-8dfe-9f5207ff1b5a)
![57ffbbe0-ec37-4e9c-a8df-d2a17497ec1f](https://github.com/user-attachments/assets/5a95a0de-be51-4b8f-bba5-96f6eea5427f)

**Comment by zhengli97** [comment]
Can you successfully run a base-to-novel experiment on you machine?

**Comment by yyjjylx** [comment]
> Can you successfully run a base-to-novel experiment on you machine?

Yes, I have run the base-to-novel experiment on my machine using six datasets referred to the article.

**Comment by zhengli97** [comment]
It looks like the code can't find the dataset. What is your current training command? Does the dataset you are currently using for the cross-dataset experiment have the same bug as the base-to-novel experiment?

**Comment by yyjjylx** [comment]
My current training command is "sh scripts/promptkd/xd_train.sh caltech101 1". 
Not yet,  the directory settings for my dataset are the same as dataset.md mentioned

**Comment by zhengli97** [comment]
That's weird. Did you change anything when you cloned the code?

**Comment by yyjjylx** [comment]
No, I followed readme.md step by step.

**Comment by zhengli97** [comment]
![image](https://github.com/user-attachments/assets/001b4d81-1600-40a4-99f9-3bcff0dc0c81)

I have downloaded the code and rerun it on my local machine. It runs succesfully. I haven't encountered the problem you mentioned. I think you can consider reconfiguring the environment and test your code again.

**Comment by yyjjylx** [comment]
Ok, thank for your patience!
